db-recipe-book/data/migrations/20181011171732_dishes_table.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('dishes', function(tbl) {
+      tbl.increments();
+      tbl.string('name', 50).notNullable();
+      tbl.integer('dish_id');
+      
+    //   tbl.integer('recipe_id').unsigned();
+      tbl.foreign('recipe_id').references('recipes.recipe_id');
```

Comment:
The foreign key reference should only be on the `recipes` table, since there are many recipes for one dish.